frontend/src/components/LoginForm.jsx

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import { useForm } from "react-hook-form";
+import { Link } from "react-router-dom";
+import logo from "../assets/images/logo.png";
+
+export default function LoginForm() {
+  const {
+    register,
+    formState: { errors },
+  } = useForm();
+
+  return (
+    <div className="flex flex-col items-center">
+      <header className="m-6">
```

Comment:
by anticipation, header must not be in a main. maybe make a component if it is present in several pages.